curvefs/src/client/rpcclient/metacache.cpp

```diff
@@ -32,6 +32,39 @@ using ::curve::common::WriteLockGuard;
 namespace curvefs {
 namespace client {
 namespace rpcclient {
+
+void MetaCache::SetTxId(uint32_t partitionId, uint64_t txId) {
+    ReadLockGuard w(txIdLock_);
+    partitionTxId_[partitionId] = txId;
+}
+
+void MetaCache::GetTxId(uint32_t partitionId, uint64_t* txId) {
```

Comment:
I think it should return bool to mark get ok or not

Reply:
> I think it should return bool to mark get ok or not

yep, i will improve it.